includes/options/class-amp-options-manager.php

```diff
@@ -562,6 +567,23 @@ public static function persistent_object_caching_notice() {
 		}
 	}
 
+	/**
+	 * Render the Stories deprecation admin notice.
+	 */
+	public static function render_stories_deprecation_notice() {
+		if ( self::is_stories_experience_enabled() ) {
+			printf(
+				'<div class="notice notice-warning"><p>%s</p></div>',
+				esc_html__( 'The Stories experience in the AMP plugin has been deprecated and will no longer be supported. A separate plugin for Stories be available soon for testing. Please backup or export your existing Stories as they will not be available in the next version of the AMP plugin.', 'amp' )
+			);
```

Comment:
This should be shown on the AMP admin settings screen as well as the Stories post list admin screen. Otherwise, it should not be shown, since it would be excessively spammy I think.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/4219/commits/ef30768fc89d12d477d27bf7a8ab6dd1e7f9cecc.